mobile/src/reducers/calendar/calendar-event.model.ts

```diff
@@ -54,4 +54,20 @@ export class CalendarEvent {
     @dataMember()
     @required()
     public status: CalendarEventStatus;
+
+    public get isRequested(): boolean {
+        return this.status === CalendarEventStatus.Requested;
+    }
+
+    public get isCompeted(): boolean {
+        return this.status === CalendarEventStatus.Completed;
+    }
+
+    public get isCancelled(): boolean {
+        return this.status === CalendarEventStatus.Cancelled;
+    }
+
+    public get isSickLeave(): boolean {
```

Comment:
is it possible to use type guards here as all statuses are type-specific?

Reply:
We have one common CalendarEvent type for all event types. If we have separate types SickLeaveEvent, VacationEvent, ... with different fields, it will be useful to use guard